lib/soap/wsdl.ex

```diff
@@ -28,21 +28,40 @@ defmodule Soap.Wsdl do
     protocol_namespace = get_protocol_namespace(wsdl)
     soap_namespace = get_soap_namespace(wsdl, opts)
     schema_namespace = get_schema_namespace(wsdl)
+    port_type = get_port_type(wsdl, protocol_namespace)
 
     parsed_response = %{
       namespaces: get_namespaces(wsdl, schema_namespace, protocol_namespace),
-      endpoint: get_endpoint(wsdl, protocol_namespace),
-      complex_types: get_complex_types(wsdl, schema_namespace, protocol_namespace),
-      operations: get_operations(wsdl, protocol_namespace, soap_namespace),
+      endpoint: get_endpoint(wsdl, protocol_namespace, soap_namespace),
+      elements: get_elements(wsdl, schema_namespace, protocol_namespace),
```

Comment:
Question, why change this name from `complex_types` to `elements`? Should we also update the name in 

https://github.com/potok-digital/soap/blob/125138c089/lib/soap/request/params.ex#L244

and

https://github.com/potok-digital/soap/blob/125138c089/lib/soap/request/params.ex#L220

Reply:
By the way, there are failing tests in master right now, something with mocks. It would be really cool to have this fixed, I would like to contribute more to this repo.

Reply:
Mocks are fixed now. You need to pull master branch :)